### ISSUE TITLE
no image for architecture 'arm64'

### ISSUE THREAD

**Issue by leolambertini** [dappnode/DAppNodePackage-bitcoin#21]
Wondering if Im doing something wrong. I get a "no image for architecture 'arm64'" message on the DAppStore while trying to find Bitcoin. 

Thank you

**Comment by alexpeterson91** [comment]
Sorry for the delayed reply, but this package was not ever designed to support ARM and uses an AMD64 binary, this was one of the first packages made and never really had much use for most dapp nodeers, as the disk space for the blockchain took up too much space to also stake ETH etc when that became the killer use case.  

A year or 2 ago i noticed there is an upstream docker image multi-architecture we can refactor this whole package into which would make it far easier to manage give us the latest info and support more thrn just AMD64 arch's, however it was never a priority for the team and i have approved, and merged PRs for years that havent been published.  

Unofrtunately only 2-3 people have pushlishing ability for official DNPs, and this DNP has no champion. 

TL;DR

Youre not doing aything wrong this build as is and has been for years is only for AMD64, needs extensive modification to support ARM64 or a full refactor (likely faster and easier) to the bitcoin-core docker image would support it all, theres just very little demand for this package comapared to the rest and dev power is stretched thin over the many many newer projects theyve been building and rolling out, this package is just considered like "legacy" by the team and will be fixed if its broken, but its very low on priority as there is 0 economic incentive from this DNP compared to aything new being  developed.  

I was personally gonna do the refactor a couple years back now but it would fundamentally change the way the package works and would remove the wallet functionality, (not RPC connections for external wallets, just the internal wallet that never should be used anyway) But got no feedback from the team or from the public on Discord so it seemed more like nobody cared at all at the time at least, nobody had room for a full BTC blockchain and fulll ETH stack +/ Gnosis stack etc, so it just was super low priority.
